src/content/docs/style-guide/quick-reference/titles.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Titles vs Short titles vs Path vs Redirects
+---
+
+This doc explains the difference between Titles and Short titles. It also delves into how these are affected and affect Paths and Redirects.
+
+## Titles
+Titles are the title rendered at the top of each doc. The title of this doc is **Titles vs Short Titles vs Path vs Redirects**. 
+
+The title of a document is define in the front matter. This doc's title is defined below:
+
+```
+---
+title: Titles vs Short Titles vs Path vs Redirects
+---
+```
+
+
+## Short titles
```

Comment:
Throughout: Do you need to add the anchor code  such as [#definition] after each H2, or do they get auto generated?